### DISCUSSION TITLE
How to remove scroll to top animation on <Link> ?

### DISCUSSION THREAD

**Discussion by da-kicks-87** [timlrx/tailwind-nextjs-starter-blog#425] [Q&A] [answered]
Hello,

When I click on a <Link> the page with go to the correct route but there is a scroll to top animation applied. I have even tried the default Next.js <Link> and still same animation is there.

 How do I remove this animation?

**Comment by timlrx** [comment, marked as answer]
Try removing [smooth-scroll](https://github.com/timlrx/tailwind-nextjs-starter-blog/blob/master/pages/_document.js#L5)

**Reply by da-kicks-87** [reply]
Great, thanks!
